flocx_market/models/offer.py

```diff
@@ -0,0 +1,53 @@
+from db import db
```

Comment:
There is no module named `db`:

```
>>> import flocx_market.models.offer
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/lars/projects/flocx/flocx-market/flocx_market/models/offer.py", line 1, in <module>
    from db import db
ModuleNotFoundError: No module named 'db'
```

Please use fully qualified imports.  Instead of:

```
from db import db
```

Use:

```
from flocx_market.db import db
```

Reply:
got it